src/Uno.UWP/Security/Authentication/Web/WebAuthenticationBrokerProvider.cs

```diff
@@ -0,0 +1,39 @@
+﻿#nullable enable
+using System;
+using System.Linq;
+
+namespace Uno.AuthenticationBroker
+{
+	public partial class WebAuthenticationBrokerProvider : IWebAuthenticationBrokerProvider
+	{
+		internal static TimeSpan Timeout => WinRTFeatureConfiguration.WebAuthenticationBroker.AuthenticationTimeout;
+
+		public Uri GetCurrentApplicationCallbackUri()
+		{
+			var defaultUri = WinRTFeatureConfiguration.WebAuthenticationBroker.DefaultReturnUri;
+
+			if (defaultUri != null)
+			{
+				// A default one is defined: we're using it instead of using
+				// a discovery mechanism.
+				return defaultUri;
+			}
+
+			var schemes = GetCustomSchemes().ToArray();
+
+			if(schemes.Length == 0)
+			{
+				throw new InvalidOperationException("No custom scheme found for this application.");
```

Comment:
```suggestion
				throw new InvalidOperationException("No custom scheme found for this application, you can define on in [CHANGEME]");
```

Reply:
The solution to this problem is platform-specific, but the message is platform-agnostic.  So it's difficult to give specific instructions to dev.